scripts/upload_to_s3.py

```diff
@@ -0,0 +1,49 @@
+"""
+Script used to upload a file to an S3 bucket.
+
+"""
+
+import boto3
+from dotenv import load_dotenv
+import argparse
+
+
+load_dotenv()
+
+BUCKET_NAME = "openearth.cap"
```

Comment:
should this also be loaded into .env ? not sure honestly if it will ever change

Reply:
Would be good to get this from env as you might want to have multiple different systems running in the same AWS account etc., so being able to change the bucket name etc. when necessary is nice.

Reply:
I moved it to .env as suggested. Thanks